jina/hubble/helper.py

```diff
@@ -250,3 +253,33 @@ def upload_file(
     response = getattr(requests, method)(url, data=data, headers=headers, stream=stream)
 
     return response
+
+
+def install_requirements(
+    requirements_file: 'Path', timeout: int = 1000, excludes: List[str] = []
+):
+    """Install modules included in requirments file
+
+    :param requirements_file: the requirements.txt file
+    :param timeout: the socket timeout (default = 1000s)
+    :param excludes: the excluded module dependencies
+    """
+
+    with requirements_file.open() as requirements_txt:
+        install_reqs = [
+            str(req)
+            for req in pkg_resources.parse_requirements(requirements_txt)
+            if req.project_name not in excludes or len(req.extras) > 0
+        ]
+
+    subprocess.check_call(
```

Comment:
isn't there a `pip` python client?

Reply:
this is officially recommended way https://pip.pypa.io/en/latest/user_guide/#using-pip-from-your-program 

Since pip v10, some api has been moved into different places.